### ISSUE TITLE
Delete call from history

### ISSUE THREAD

**Issue by crookedbard** [stephenlb/webrtc-sdk#64]
It is possible to get phone call history from pubnub. But how to delete the phone call from history manually?

There is an option in pubnub admin portal:
![image](https://user-images.githubusercontent.com/4722359/50086315-48253200-0205-11e9-89de-816771243df8.png)

I understand that it will store the phone call to history for 7 days and there should be a way to delete it manually ?

**Comment by stephenlb** [comment]
yes this is available now.  Follow the [History Fetch](https://www.pubnub.com/docs/pubnub-rest-api-documentation#history-fetch-history-get) and [Delete](https://www.pubnub.com/docs/pubnub-rest-api-documentation#from-history-delete) API documents:
